rhv_checks.py

```diff
@@ -338,6 +340,54 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def is_service_active(ssh, name):
+    """helper function for check_services_running"""
+    stdin, stdout, stderr = ssh.exec_command("systemctl status {}".format(name))
+    status = stdout.read()
+    return "Active: active (running)" in status
+
+
+def check_services_running(system):
+    """Check to see if the vdsm, ovirt-ha-agent, ovirt-ha-broker service is up/running"""
+    hosts = system.api.system_service().hosts_service()
+    hosts_agents = dict()
+    hosts_status = dict()
+    for host in hosts.list():
+        host_service = hosts.host_service(host.id)
+        nics = host_service.nics_service().list()
+        ip_addresses = [n.ip.address for n in nics if n.ip.address.startswith('10.')]
```

Comment:
Please mention 
https://github.com/paramiko/paramiko/issues/686
https://github.com/paramiko/paramiko/issues/770

in comments for this line as I think, based on my quick glance, Paramiko does not support specifying use of IPv4/IPv6.

Reply:
@kedark3 sure, that's exactly the reason I had to do this workaround